erpnext/manufacturing/doctype/work_order/work_order.json

```diff
@@ -110,7 +110,8 @@
    "fieldname": "item_name",
    "fieldtype": "Data",
    "label": "Item Name",
-   "read_only": 1
+   "read_only": 1,
+   "unique": 1
```

Comment:
Wouldn't this not only users to create a second work order for the same item, now that it's unique?